src/engraving/dom/check.cpp

```diff
@@ -158,9 +158,9 @@ Ret Score::sanityCheckLocal()
             Rest* fmrest0 = nullptr; // full measure rest in voice 0
             Fraction voices[VOICES];
 
-#ifndef NDEBUG
+// #ifndef NDEBUG
```

Comment:
I'd rather remove them than comment them

Reply:
There were some (very) few similar ones, should I remove them too?

Reply:
I'd leave those that are not related to corrupted measures untouched